dev/com.ibm.ws.jdbc.4.3/src/com/ibm/ws/rsadapter/jdbc/v43/WSJdbc43CallableStatement.java

```diff
@@ -39,27 +41,63 @@ public WSJdbc43CallableStatement(CallableStatement cstmtImplObject, WSJdbcConnec
     public String enquoteLiteral(String val) throws SQLException {
         // KEEP CODE IN SYNC: This method is duplicated in WSJdbc43Statement, WSJdbc43PreparedStatement,
         // and WSJdbc43CallableStatement because multiple inheritance isn't allowed.
-        throw new UnsupportedOperationException();
+        try {
+            return cstmtImpl.enquoteLiteral(val);
+        } catch (SQLException ex) {
+            FFDCFilter.processException(ex,
+                                        "com.ibm.ws.rsadapter.jdbc.v43.WSJdbc43CallableStatement.enquoteLiteral(String)", "48", this);
+            throw WSJdbcUtil.mapException(this, ex);
+        } catch (NullPointerException nullX) {
+            // No FFDC code needed; we might be closed.
+            throw runtimeXIfNotClosed(nullX);
+        }
     }
 
     @Override
     public String enquoteIdentifier(String identifier, boolean alwaysQuote) throws SQLException {
         // KEEP CODE IN SYNC: This method is duplicated in WSJdbc43Statement, WSJdbc43PreparedStatement,
         // and WSJdbc43CallableStatement because multiple inheritance isn't allowed.
-        throw new UnsupportedOperationException();
+        try {
+            return cstmtImpl.enquoteIdentifier(identifier, alwaysQuote);
+        } catch (SQLException ex) {
+            FFDCFilter.processException(ex,
+                                        "com.ibm.ws.rsadapter.jdbc.v43.WSJdbc43CallableStatement.enquoteIdentifier(String, String)", "64", this);
```

Comment:
Misleading method signature (String,String), when the actual signature is (String, boolean).
Also note that common practice for FFDC is to avoid the method name and signature altogether and just do
FFDCFilter.processException(ex, getClass().getName(), "64", this);

Reply:
Good catch.